src/dotnet/Program.cs

```diff
@@ -114,6 +114,8 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
                 }
                 else
                 {
+                    firstTimeUseSentinelPath = ConfigureDotNetForFirstTimeUse();
```

Comment:
@livarcocc I think that there was a disconnect somewhere. Let's chat tomorrow in person on this. It got merged now, so if it is to be fixed, it needs to be fixed in the next release.